### DISCUSSION TITLE
Adjust Interval remotely

### DISCUSSION THREAD

**Discussion by iridium2001** [jomjol/AI-on-the-edge-device#2221] [Ideas]
I would like a feature that can set the scan interval remotely, e.g. from Home Assistant.  

During the day, 5 min work just fine for me. However, when I'm on vacation or at work or even during night hours, the interval could be every 60 or 120 minutes. 
If this could be remotely set, it could be automated and would not even require additional logic or scheduling in the device.

**Comment by caco3** [comment]
What is the benefit for it?

If you want to reduce the amount of data in a database, you should do this in post processing.

**Comment by iridium2001** [comment]
Perhaps I'm just too unexperienced in the database setup (any hint for a good read about this is appreciated :-) ), but for me it seems easier to not create unneccesary data in the first place. 

If the interval could be set via the API, this would be straight forward.

**Comment by caco3** [comment]
see the comments in https://github.com/jomjol/AI-on-the-edge-device/issues/2226
